tests/Unit/Domain.LinnApps.Tests/PurchaseOrderServiceTests/WhenUpdatingPurchaseOrderConvValueQtyFields.cs

```diff
@@ -395,5 +392,35 @@ public void ShouldUpdateMiniOrderFields()
             this.miniOrder.OrderTotal.Should().Be(19862.33m);
             this.miniOrder.BaseOrderTotal.Should().Be(24827.91m);
         }
+
+        [Test]
+        public void ShouldUpdateDeliveryTotalFields()
+        {
+            this.current.OrderNumber.Should().Be(600179);
+            this.current.Remarks.Should().Be("updated remarks");
+
+            var firstDetail = this.current.Details.First();
+
+            var delivery = firstDetail.PurchaseDeliveries.First();
+
+            // order unit price currency, our unit price currency, our delivery qty
+            delivery.OrderUnitPriceCurrency.Should().Be(120m);
```

Comment:
You didn't change this away from 120m in these tests so this isn't testing the delivery price was updated